dlib/media/ffmpeg_demuxer.h

```diff
@@ -995,7 +996,6 @@ namespace dlib
             Callback&& clb
         )
         {
-            using namespace std::chrono;
```

Comment:
Same thing here, just do `using std::chrono::whatever` to bring in the symbols needed.  Name lookup is such that that doesn't become ambiguous when people do funny things ahead of the code.